st2common/bin/st2ctl

```diff
@@ -76,8 +74,10 @@ function st2stop() {
 function service_manager() {
   if [ -d /run/systemd/system ]; then
     systemctl "${2}" "${1}"
-  elif command -v service 2>/dev/null; then
+  elif command -v service > /dev/null 2>&1; then
     service "${1}" "${2}"
+  elif [ -x /etc/init.d/${1} ]; then
```

Comment:
ack, that you got it? Basically system might have no service binary, and you are obliged to support upstart anyway.